src/app/modules/core/core.module.ts

```diff
@@ -82,7 +83,10 @@ import {BootProgressService, BootProgressServiceFactory} from './services/boot-p
             debounceTime: 50
         }),
         NgProgressHttpModule,
-        NgProgressRouterModule
+        NgProgressRouterModule,
+
+        // ng2-charts module
+        ChartsModule
```

Comment:
I don't believe you need this imported here since it's not a globally required module - it's used only in the `dashboard`. :)

Reply:
Tried to do so but the page stopped working as a result, so I left it there

Reply:
You can safely remove the `ChartsModule` here (as well as the import). This needs to be imported only in the `dashboard.module.ts` since it's used only there.